config/main.py

```diff
@@ -14,7 +14,7 @@
 import ipaddress
 from swsssdk import ConfigDBConnector, SonicV2Connector, SonicDBConfig
 from minigraph import parse_device_desc_xml
-from config_mgmt import ConfigMgmt, ConfigMgmtDPB
```

Comment:
When running `config` command on the switch, appears next traceback
`  File "/usr/lib/python2.7/dist-packages/config/main.py", line 17, in <module>`
`  from config_mgmt import ConfigMgmtDPB` 
`   ImportError: No module named config_mgmt`

Reply:
@samaity / @praveen-li : Can you please investigate this?

Reply:
@praveen-li any idea? btw I am getting another error like _**"ImportError: No module named jsondiff - required module not found"**_ in the test case running for the PR. I believe we have already installed jsondiff. Right @praveen-li ? Am I missing something?

Reply:
@smaity: Kindly point me to the test results. This should not happen now on Azure Jenkins..

Reply:
@praveen-li, I am seeing this error [here](https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/sonic-utilities-build-pr/1753/testReport/junit/test_crm/TestCrm/test_CrmIpv6Route/)